### ISSUE TITLE
$CONF not found.

### ISSUE THREAD

**Issue by PixellUp** [jgmdev/ddos-deflate#32]
After installing everything successfully, `ddos` command return following error: `$CONF not found.`
Everything that `ddos` command return:
```
/usr/local/ddos/ddos.sh: 25: [: /etc/ddos/ddos.conf: unexpected operator
DDoS-Deflate version 0.9
Copyright (C) 2005, Zaf <zaf@vsnl.com>

$CONF not found.
```
Running on **Ubuntu 16.10 x32** with **ddos-deflate v0.9**
Notes:
- Previously did not have any problem installing and running ddos-deflate
- I've reinstalled multiple times with no success

**Comment by jgmdev** [comment]
it seems that the latest commit that added freebsd support which modified the shebang from #!/bin/bash to #!/bin/sh affected ubuntu because some of the code syntax isn't supported by the shell, can you check the sh version on the root account by running:
```
sh -v
```
As a workaround, after installing ddos you can modify the ddos.sh script shebang line to point into /bin/bash

**Comment by doreto** [comment]
I'm sorry for writing this to late, but the fix worked. Thank you. Also should be there a way to check if you are running on ubuntu or freesd before installing to prevent such issue as I'm using this package on multiple servers and I must have my own copy with that fix instead of pulling direct from git.

**Comment by Mauler7** [comment]
"As a workaround, after installing ddos you can modify the ddos.sh script shebang line to point into /bin/bash"

What si the "shebang line"

I cannot find your point of reference. Please advise.

**Comment by Poil** [comment]
```
update-alternatives --install /bin/sh sh /bin/bash 100
```

**Comment by jgmdev** [comment]
Fixed on #40
